docs/theme/js/nice-select.min.js

```diff
@@ -4,4 +4,119 @@
 /*  jQuery Nice Select - v1.0
```

Comment:
OK, now I'm convinced; don't auto-format minified code. I think you can exclude _docs/theme_ (though I also think a lot of this stuff should not be checked in.)

Reply:
OK that convinced me too.